compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Repl.hs

```diff
@@ -419,6 +412,64 @@ runRepl importPkgs opts replClient logger ideState = do
             Left err -> liftIO $ renderError dflags err
             Right () -> pure ()
 
+    mkReplOption
+        :: (DynFlags -> [String] -> ExceptT Error ReplM ())
+        -> [String] -> ReplM ()
+    mkReplOption option args = do
+        ReplState {lineNumber} <- State.get
+        dflags <- liftIO $
+            hsc_dflags . hscEnv <$>
+            runAction ideState (use_ GhcSession $ lineFilePath lineNumber)
+        r <- runExceptT $ option dflags args
+        case r of
+            Left err -> liftIO $ renderError dflags err
+            Right () -> pure ()
+    replOptions :: [(String, [String] -> ReplM ())]
+    replOptions =
+      [ ("help", mkReplOption optHelp)
+      , ("module", mkReplOption optModule)
+      ]
+    optHelp _dflags _args = liftIO $ T.putStrLn $ T.unlines
+      [ " Commands available from the prompt:"
```

Comment:
```suggestion
      [ "Commands available from the prompt:"
```
(unless I’m missing something and this is helpful for formatting)

Reply:
I followed GHCi's example here which indents the help message by one column. Presumably as a guide for the eye. Happy to remove the indent if you prefer.

Reply:
Makes sense, let’s keep it.